server/routes/api.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const router = express.Router();
+const moment = require("moment");
+const City = require("../models/City");
+const axios = require("axios").default;
+
+router.get("/city/:cityName", async (req, res) => {
+  const { cityName } = req.params;
+  const url = `http://api.openweathermap.org/data/2.5/weather?${cityName}&appid=2a3860cbf232b1d4e31f20174b149693&units=metric`;
+  try {
+    let response = await axios.get(url);
+    response = response.data
+    const dateString = moment.unix(response.dt).format("llll");
+    const cityWeather = {
+      name: response.name,
+      temperature: response.main.temp,
+      condition: response.weather[0].main,
+      conditionPic: response.weather[0].icon,
+      updatedAt: dateString
+    };
+    res.send(cityWeather);
+  } catch (error) {
+      console.log(error)
+    res.end("error");
```

Comment:
you should `res.send(error)` `res.end()` will not send a message. 

Do not send a string `'error'` you should send the actual error to the client side